### ISSUE TITLE
How to set variable "includeSoftConstraints" for Toplology Spread Constraint Violations

### ISSUE THREAD

**Issue by gallowaystorm** [kubernetes-sigs/descheduler#942]
I am looking at the Helm chart provider for this application.  On the ReadMe, it says there is an option to set "includeSoftConstraints" to true (if wanted) for Toplology Spread Constraint Violations.  How can this be done on the Helm chart?  I don't see an option for this.

**Comment by damemi** [comment]
The strategy config settings are set through the [`deschedulerPolicy` value in the Helm chart](https://github.com/kubernetes-sigs/descheduler/blob/e8fae9a/charts/descheduler/values.yaml#L61). This uses the same config as in the readme, so for example:

```yaml
strategies:
  "RemovePodsViolatingTopologySpreadConstraint":
     enabled: true
     params:
       includeSoftConstraints: false
```

**Comment by gallowaystorm** [comment]
Thank you:

https://github.com/kubernetes-sigs/descheduler/pull/943

**Comment by a7i** [comment]
Fixed in #943

/close
